Patch-Source: https://github.com/sabotage-linux/gettext-tiny/commit/bcb04de5afcf5532a110babfe3912b5725e7146b
From bcb04de5afcf5532a110babfe3912b5725e7146b Mon Sep 17 00:00:00 2001
From: Patrycja <github@ptrcnull.me>
Date: Wed, 9 Mar 2022 15:42:21 +0100
Subject: [PATCH] xgettext: fix --version output (#60)

"\n" in the printf parameter doesn't expand
and the comma gets printed after format string
---
 src/xgettext.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/xgettext.sh b/src/xgettext.sh
index fc38369..b113c98 100755
--- a/src/xgettext.sh
+++ b/src/xgettext.sh
@@ -16,7 +16,7 @@ syntax() {
 }
 
 show_version() {
-	printf "%s\n", "xgettext (GNU gettext-tools compatible) 99.9999.9999\n"
+	printf "%s\n" "xgettext (GNU gettext-tools-compatible) 99.99"
 	exit 0
 }
 
